data/input_files/tests/solid/qs_linear.lua

```diff
@@ -0,0 +1,70 @@
+-- Comparison information
+expected_x_l2norm = 2.4276439
+epsilon = 0.0001
+
+-- Simulation time parameters
+dt      = 1.0
+
+main_mesh = {
+    type = "file",
+    -- mesh file
+    mesh = "../../../meshes/beam-hex.mesh",
+    -- serial and parallel refinement levels
+    ser_ref_levels = 1,
+    par_ref_levels = 0,
+}
+
+-- Simulation output format
+output_type = "VisIt"
+
+-- Solver parameters
+solid = {
+    stiffness_solver = {
+        -- Use a direct solver to check for machine precision convergence in one Newton step
+        linear = {
+            type = "direct",
+            direct_options = {
+                print_level = 0,
+            },
+        },
+
+        nonlinear = {
+            rel_tol     = 1.0e-3,
+            abs_tol     = 1.0e-6,
+            max_iter    = 5000,
+            print_level = 1,
+        },
+    },
+
+    -- polynomial interpolation order
+    order = 1,
+
+    -- neo-Hookean material parameters
+    mu = 0.25,
```

Comment:
Presumably when we actually have a "material model module" this will get fleshed out...